app/models/user.rb

```diff
@@ -2,5 +2,6 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
-  belongs_to :business
+  belongs_to :business, required: false
```

Comment:
`required: true` is from Rails 4.  As of Rails 5, `required: true` is the default, and if you want to change it you add `optional: true`.  I don't know why they changed the keyword but I believe that's what it is, and that's what I'm using in my Rails 5 app.